docs/decisions/0037-rag-architecture-implementation.md

```diff
@@ -10,13 +10,14 @@ parent: Decision Records
 The current trend in questions and answering (Q&A) using large language models (LLMs) or other
 AI related technology is retrieval-augmented-generation (RAG).
 
-RAG is related to [Open Generative QA](https://huggingface.co/tasks/question-answering) 
+RAG is related to [Open Generative QA](https://huggingface.co/tasks/question-answering)
 that means LLM (which generates text) is supplied with context (chunks of information extracted
 from various sources) and then it generates answer.
 
 RAG architecture consists of [these steps](https://www.linkedin.com/pulse/rag-architecture-deep-dive-frank-denneman-4lple) (simplified):
 
 How source data is processed:
+
```

Comment:
For markdown install markdownlin plugin for VSCode, highlights the errors and you can fix most of them per click